observe_export/observe_export.py

```diff
@@ -725,6 +725,10 @@ def load_export_config_from_file(file: Path) -> ExportConfig:
         export_config.datasource = DataSourceWorksheet.from_dict(export_config.datasource)
     else:
         raise ValueError(f"Could not resolve the following data source: {export_config.datasource}")
+    
+    if export_config.user == None: 
+        export_config.user = export_config.url.rsplit('.', n=2)[-3]
```

Comment:
Why do a right split and not a regular (left) one? The customer ID is always the first element separated by ".". I don't think the above would work for https://CUSTOMER.XYZ.observeinc.com. Please revise.

Reply:
Also: n=2 doesn't work on my laptop. https://docs.python.org/3.8/library/stdtypes.html specifies maxsplit as optional argument. (3.9 does so as well, as well as 3.11). Not sure how this could have worked on your local machine. Please test such changes very carefully.

Reply:
Also the rsplit doesn't work even if the url doesn't contain a subdomain:
`foo = "https://CUSTOMER.observeinc.com"
foo.rsplit(".", maxsplit=2)[-3]`
outputs 'https://CUSTOMER'

Reply:
It would also be good if you could add to the docs that the url is expected to **not** contain the https://

Reply:
1. I thought that the customer ID would be then `CUSTOMER.XYZ` but you are right. I'll fix that. 
2. This is again a problem of the pandas version, not the python version. I will add a fix in the setup to make it install 1.x
3. Good point. I will add it.

Reply:
re 2: The url is a plain string, right? How could it be pandas related?

Reply:
You are right. n=2 doesn't work. I think I reinstalled observe-export at one point and forget the -e option in pip. So I wrongly thought it works. Now I tested also that the code was actually executed.